pkg/plugin/predicates/predictor.go

```diff
@@ -278,8 +283,10 @@ func (p *Predictor) predicatesAllocate(pod *v1.Pod, meta predicates.PredicateMet
 			}
 
 			if !fit {
-				events.GetRecorder().Eventf(pod, v1.EventTypeWarning,
-					"FailedScheduling", "%v", reasons)
+				if isAllowed := limiter.AllowN(time.Now(), 1); isAllowed {
```

Comment:
This can be simplified to just become `limiter.Allow()`

Reply:
OK, Sure.